src/App.java

```diff
@@ -2,30 +2,209 @@
 
 public class App {
     
-    /**
-     * This method is the primary driver method for your program.
-     * It should primarily call other methods to do its work.
-     * Do NOT edit this method until after you have fully satisfied the reqs for a 5
-     */
 
     public void start() {
         String userName = getUserName();
         printGreeting(userName);
-        printInstructions();
-        doMadlib();
-        System.out.println("hi");
+        do{                                        // loop logic learned from https://www.geeksforgeeks.org/loops-in-java/
+            int madlibChoice = chooseMadlib();
+            if (madlibChoice == 1){
+                printInstructions();
+                doMadlib1(); 
+            }
+            else if (madlibChoice == 2){
+                printInstructions();
+                doMadlib2();                        // I tried submiting it yesterday but had some issues with the commit part. 
+            }                                       // thanks to joão i was able to fix it. sorry for the late submission.
+            else {
+                errorMessage();
+            }
+        } while(restart() == true); 
+        endMessage();   
     }
     
-    /* Here you should create all missing methods used by the start() method
-     * I have created the signature for ONE of the methods below.
-     * You can create any other methods you need as well!
-     */ 
-    
-    public void printInstructions() {
+
+    public String getUserName (){
+        Scanner getName = new Scanner(System.in);
+        System.out.println ("Enter your name below:");
+        return getName.nextLine();
+    }
+
+    public void printGreeting(String userName){
+        System.out.println("Hello " + userName + ", welcome to Madlibs!");
+    }
+
+    public void errorMessage(){
+        System.out.println ("This madlib does not exist :(");
+    }
+
+    public int chooseMadlib(){
+        Scanner getChoice = new Scanner (System.in);
+        System.out.println();
+        System.out.println("Enter 1 or 2 to choose a Madlib:");
+        return getChoice.nextInt();    
+    }
+
+    public boolean restart(){
+        Scanner getRestart = new Scanner(System.in);
+        System.out.println();          
+        
+        //  The following code was copied from chat gpt with some minor alterations
```

Comment:
I asked you to link to a transcript for any AI. With chatgpt there is a share button that gives access to a live transcript of the chat, I'd love for you to send that to me via email if you still can.